src/app/rpmostree-dbus-helpers.c

```diff
@@ -29,6 +29,26 @@
 #include <signal.h>
 #include <systemd/sd-login.h>
 
+static gboolean opt_txn_quiet;
+
+static const GOptionEntry txn_common_option_entries[] = {
+  { "quiet", 'q', 0, G_OPTION_ARG_NONE, &opt_txn_quiet, "Quiet output; Primarily, do not emit progress information", NULL },
```

Comment:
Maybe let's split this instead into two options: `--no-progress` and `--quiet`? I think what is here right now corresponds directly to `--no-progress`. And a `--quiet` would additionally discard `Message`/`Task`?